PATTERNS.md

```diff
@@ -1179,3 +1180,422 @@ e.g.
   </div>
 </div>
 {{ end }}
+```
+
+## Fields
+
+The field components provides a consistent way of adding a simple valid html input element with corresponding field validation by populating relevant properties on the model.
+
+Fields available:
+
+- [text](#text-input)
+- [textarea](#textarea)
+- [radios fieldset](#radios-fieldset)
+
+The inputs contained within the fields share a common model `input.go`, this allows [attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes) to be set in a consistent way. However, there are circumstances where the attributes are not permitted and subsequently not rendered. Use the table below as a guide:
+
+| Attribute                                                                                 | Checkbox | Radio | Text | Textarea |
+| ----------------------------------------------------------------------------------------- | -------- | ----- | ---- | -------- |
+| [Autocomplete](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) | No       | No    | Yes  | Yes      |
+| ID (mandatory)                                                                            | Yes      | Yes   | Yes  | Yes      |
+| Checked                                                                                   | Yes      | Yes   | No   | No       |
+| Disabled                                                                                  | Yes      | Yes   | Yes  | Yes      |
+| Name (mandatory)                                                                          | Yes      | Yes   | Yes  | Yes      |
+| Value                                                                                     | Yes      | Yes   | Yes  | Yes      |
+
+### Text input
+
+To instatiate the [text input](https://service-manual.ons.gov.uk/design-system/components/input) UI component in your service:
```

Comment:
'instantiate'

Reply:
🤦‍♂️ - I changed this everywhere else but obvo not from the bit I copied and then pasted 🚕

Reply:
It's the only reason I noticed it tbf.